DependencyInjection/FreshDoctrineEnumExtension.php

```diff
@@ -29,5 +29,9 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new YamlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.yml');
+
+        if (\class_exists('Symfony\Bridge\Doctrine\Form\DoctrineOrmTypeGuesser')) {
```

Comment:
This condition is incorrect. You pretty much always use symfony/doctrine-bridge even if you don't use symfony/form. So this class will "exist". By "exist" I mean that loading it will cause a fatal error because `Symfony\Component\Form\FormTypeGuesserInterface` will not exist. So here you should check for the `FormTypeGuesserInterface`.

Reply:
Also you can use `::class`.

Reply:
@fre5h I fixed it for you. https://github.com/fre5h/DoctrineEnumBundle/pull/124